common/static/common/js/discussion/main.js

```diff
@@ -27,13 +27,10 @@
                     sort: sort_preference
                 });
                 course_settings = new DiscussionCourseSettings(element.data("course-settings"));
-                // suppressing Do not use 'new' for side effects.
-                /* jshint -W031*/
-                new DiscussionRouter({
+                new DiscussionRouter({  // eslint-disable-line no-new
```

Comment:
Just for the record, I'd prefer that routers not auto-start, but that instead they have explicit calls to start and stop them. No change needed for now...